src/helpers/get-custodian-folder-creation-cache-key.helper.test.ts

```diff
@@ -0,0 +1,30 @@
+import { CUSTODIAN } from '@ukef/constants';
+import { RandomValueGenerator } from '@ukef-test/support/generator/random-value-generator';
+
+import { getCustodianFolderCreationCacheKey } from './get-custodian-folder-creation-cache-key.helper';
+
+describe('GetCustodianFolderCreationCacheKey helper test', () => {
+  const valueGenerator = new RandomValueGenerator();
+
+  describe('getCustodianFolderCreationCacheKey', () => {
+    const parentFolderId = valueGenerator.nonnegativeInteger();
+
+    it('test cache key generation without reimplementing full method', () => {
+      const key1 = getCustodianFolderCreationCacheKey(parentFolderId, valueGenerator.string());
```

Comment:
For the instances where we have `valueGenerator.string()` inline as the second argument, can we pull these out into variables (e.g., `folderName`, `anotherFolderName`) for readability

Reply:
Thank you, used variables to make calls more readable.